packages/account/src/Components/poi/status/unsupported/card-details/uploader.tsx

```diff
@@ -45,58 +60,62 @@ const Preview = ({ data, setFieldValue, value, has_frame, handleChange }) => {
                 style={{ backgroundImage: `url(${background_url})` }}
             >
                 {has_frame && <Icon icon='IcPoiFrame' className={`${ROOT_CLASS}__uploader-frame`} />}
-                {(!background_url || value.file.type.indexOf('pdf') !== -1) && (
+                {(!background_url || value?.file.type.indexOf('pdf') !== -1) && (
                     <React.Fragment>
                         <Icon icon='IcCloudUpload' size={50} />
                         <Text as='p' size='xs' color='general' align='center'>
-                            {value.file.name}
+                            {value?.file.name}
                         </Text>
                     </React.Fragment>
                 )}
                 <Icon
                     icon='IcCloseCircle'
                     className={`${ROOT_CLASS}__uploader-remove`}
-                    onClick={() => handleChange(null, setFieldValue)}
+                    onClick={() => {
+                        handleChange?.(null, setFieldValue);
+                    }}
                     size={16}
                 />
             </div>
             <Text as='p' size='xs' color='general' align='center'>
-                {data.info}
+                {data?.info}
             </Text>
         </div>
     );
 };
 
-const Uploader = ({ data, value, is_full, onChange, has_frame }) => {
-    const [image, setImage] = React.useState();
+const Uploader = ({ data, value, is_full, onChange, has_frame }: Partial<TUploader>) => {
+    const [image, setImage] = React.useState<FormikValues>();
 
     React.useEffect(() => {
         setImage(value);
     }, [value]);
 
-    const handleChange = (file, setFieldValue) => {
+    const handleChange = (file?: object, setFieldValue?: FormikProps<FormikValues>['setFieldValue']) => {
         if (onChange && typeof onChange === 'function') {
             onChange(file);
         }
```

Comment:
```suggestion
       onChange?.(file);
```

Reply:
we define type of onChange function, there is no necessity to check, if it is function